includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2580,18 +2586,24 @@ public function get_icon_url() {
 	 * Supported: statement_descriptor, business_name, business_url, business_support_address,
 	 * business_support_email, business_support_phone, branding_logo, branding_icon,
 	 * branding_primary_color, branding_secondary_color.
+	 *
+	 * $return array | WP_Error Update account result.
+	 *
+	 * @throws Exception
 	 */
 	public function update_account( $account_settings ) {
 		if ( empty( $account_settings ) ) {
 			return;
 		}
 
-		$error_message = $this->account->update_stripe_account( $account_settings );
+		$request = $this->account->update_stripe_account( $account_settings );
```

Comment:
Just a suggestion but I feel since we're anyways catching the exception in the function below, we could potentially return a consistent response in both success and failure scenarios. Something like:
```
{
    "updated": "false",
    "error_message": "<// $e->getMessage() >" 
}
```
And then in the calling function, check for the `boolean` to decide what to log. Also makes the naming convention a lot simpler so instead of `request`, it can be `stripe_account_updation_response` which can then handle both cases.

In either case the naming does seem off for this, so that can be improved regardless.